Gemfile

```diff
@@ -1,6 +1,11 @@
 source 'http://rubygems.org'
 
-case version = ENV['MONGOID_VERSION'] || '~> 4.0'
+gemspec
+
+case version = ENV['MONGOID_VERSION'] || '~> 5.0'
+when /5/
+  gem 'mongoid', '~> 5.0'
+  gem 'mongoid-slug', github: 'dblock/mongoid-slug', branch: 'mongoid-5'
```

Comment:
Until there's a mongoid 5 compatible version of mongoid-slug, see https://github.com/digitalplaywright/mongoid-slug/pull/187